source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -324,9 +325,14 @@
                                 return valueItem;
                             }),
                             comments: _.map(commentsByKeyCollection, function (commentsByKey) {
-                                var commentItem = GetCommentItem(commentsByKey.commentRow.calificationId, commentsByKey[item.Key], commentsByKey.commentRow.evaluatorEmployee, commentsByKey.commentRow.evaluatorEmployee == self.userLogged)
+                                var commentItem = GetCommentItem(
+                                    commentsByKey.calificationColumn.calificationId,
+                                    commentsByKey[item.Key],
+                                    commentsByKey.calificationColumn.evaluatorEmployee,
+                                    commentsByKey.calificationColumn.evaluatorEmployee == self.userLogged)
 
                                 commentItem.IsEditingComment = ko.observable(false);
+                                commentItem.isEditable = self.isValueEditable(commentsByKey);
```

Comment:
this is complex so Andres an I work together to understand it better. It works fine but this is a bit stinky, so we probably need to rework all this logic